foundation-config/src/main/java/io/servicecomb/config/archaius/sources/AbstractConfigLoader.java

```
@@ -54,9 +57,9 @@ public ConfigModel load(URL url, String resourceName) throws IOException {
 
         ConfigModel configModel = new ConfigModel();
         configModel.setUrl(url);
-        //        if (resourceName != null) {
-        //            configModel.setRootPath(FileUtils.findRootPath(url, resourceName));
-        //        }
+        if (resourceName != null) {
```

If the resourceName is null ,  it's better to throw the exception here.